### PR TITLE
changes and updates according new design of some components

### PR DESCRIPTION
**Pull request by netergart**
### Added
 - toggle component  via @fstasi 
 
 
 ### Fixed
 - inputs and select styles
 - fixed reset method and allow default HTML5 attributes in the input
 - allow using children components inside radio/checkbox components
 
  
 ### Changed
 - updated checkbox UI color
 - added more style variants in the select component
- updated PopoverTrigger exports
 
 P.S.
This can be big PR because I transferred all changes from the old repo and I need all changed inside classroom #4
All next PR will be correct 1 feature for 1 PR

### CODE REVIEW

**Comment by Serjlee** [comment]
@fstasi have you the chance to review this in the near future?

**Review by fstasi — CHANGES_REQUESTED** [review]
I'd go with react 16 if possible. the pr overall is good

File: src/components/checkbox/CheckboxGroupItem.tsx [hunk 1]
```diff
@@ -74,6 +74,7 @@ export function CheckboxGroupItem({
     <RadioCheckboxWrapper {...CheckboxWrapperProps}>
       <input {...inputProps} className={inputClasses} ref={ref} />
       <span className={checkboxClasses}></span>
+      {props.children}
```
[the remark this answers is not in the document]
Comment (lukdog) [reply src/components/checkbox/CheckboxGroupItem.tsx]: Fixed in [b81e445](https://github.com/arduino/arc/pull/3/commits/b81e4453186fb68a69e5ea7092129a583860822f)

File: src/components/checkbox/RadioGroupItem.tsx [hunk 2]
```diff
@@ -68,6 +68,7 @@ export function RadioGroupItem({
     <RadioCheckboxWrapper {...CheckboxWrapperProps}>
       <input {...inputProps} className={inputClasses} ref={ref} />
       <span className={radioClasses}></span>
+      {props.children}
```
[the remark this answers is not in the document]
Comment (lukdog) [reply src/components/checkbox/RadioGroupItem.tsx]: Fixed in [b81e445](https://github.com/arduino/arc/pull/3/commits/b81e4453186fb68a69e5ea7092129a583860822f)

**Review by fstasi — APPROVED** [review]

**Pull request merged**